client/jetpack-onboarding/steps/summary.jsx

```diff
@@ -18,6 +18,8 @@ import DocumentHead from 'components/data/document-head';
 import FormattedHeader from 'components/formatted-header';
 import PageViewTracker from 'lib/analytics/page-view-tracker';
 import Spinner from 'components/spinner';
+import { addCalypsoEnvQueryArg } from 'jetpack-connect/utils';
+import { REMOTE_PATH_AUTH } from 'jetpack-connect/constants';
```

Comment:
It may not be a good idea to import from these files. We may need a solution for where to keep Jetpack functions, constants, etc., which are shared across sections.